### ISSUE TITLE
Dotazy na solr

### ISSUE THREAD

**Issue by annie-cz** [ceskaexpedice/kramerius-admin-client#16]
Bylo by možné v budoucnu zahrnout do admin klienta i dotazy na solr?

Typické příklady z praxe:

Jaká uuidčka jsou v dané sbírce a kolik jich je?
Najdu si autora Karel Čapek ==> potřebuji všechna uuid, kde tento autor figuruje jako autor.
Najít všechny uuidčka všech monografií, jejichž datum vydání je od roku 2000 do roku 2001.
Potřebuji najít všechna uuidčka, která doposud nemají přiřazený donátor (např. ve sbírce nebo i z jiného výběru)

Atd. atd. ...

Výstupy by byly ideální jako seznam, který by si admin sám modifikoval. Tj. minimum vidím jako seznam uuid, ve volitelném případě např. uuid, název, datum vydání, příznak, licence, autor, atd.

Výstupy ve formě seznamu uuid jsou super z důvodu, že s nimi pak kurátor může rovnou začít dělat další akci => např. zvolený seznam vzít a rovnou ho opravit.

**Comment by honza-rychtar** [comment]
Ano. Otázka je, jestli to má být součástí Admin rozhraní. 

Jsou to vše veřejná data a tohle by uvítali nejen ti, kteří mají přístup do admin rohzraní.

**Comment by annie-cz** [comment]
Souhlasím, že to klidně může být i v klientovi. Akorát to chce, aby pak klient uměl zpracovat "select all", protože představa, že si něco takového vyfiltruju a pak to mám klikat ručně po jednotlivých kusech, už tak krásná není. :)

**Comment by pavel-stastny** [comment]
Bude soucasti noveho klienta a pripadne v systemu Anakon
